controllers/lighting_controller/lighting_controller.py

```diff
@@ -71,14 +75,25 @@ class LuminosityFade:
 CUE_STACK = [
     LightingEffect(
         0,
-        lighting=[ArenaLighting('SUN', intensity=1)],
+        lighting=[ArenaLighting('SUN', intensity=NON_MATCH_LIGHTING_INTENSITY)],
         luminosity=0.05,
         name="Pre-set",
     ),
-    LightingEffect(0, fade_time=1.5, name="Fade-up"),
+    LightingEffect(
+        0,
+        fade_time=1.5,
+        lighting=[ArenaLighting('SUN', intensity=MATCH_LIGHTING_INTENSITY)],
+        name="Fade-up",
+    ),
     LightingEffect(
         -0.1,
-        lighting=[ArenaLighting('SUN', intensity=1, colour=(1, 0, 0))],
+        lighting=[
+            ArenaLighting(
+                'SUN',
+                intensity=NON_MATCH_LIGHTING_INTENSITY,
```

Comment:
You need a non-zero intensity at the end so that the red colour shows.

Reply:
https://github.com/srobo/competition-simulator/pull/365/commits/914c35b6ae343913d7976d713d10132e640720d7

Reply:
https://github.com/srobo/competition-simulator/pull/365/commits/8e0c48ea41ec93754aca23bc9a1bca570063adc4

Reply:
I think this is resolved now? #379 seemed to help a lot.